### PR TITLE
runtimeclass: update type description to beta

### LINKED ISSUE #76970: runtimeclass: type description indicates alpha - should be beta

### PR DESCRIPTION
**Pull request by egernst**
**What type of PR is this?**
/kind documentation

**What this PR does/ why we need it** 

The existing runtimeClass description indicated this was an alpha feature. Since this moved to beta, let's go ahead and update. No functional changes are introduced.

Fixes: https://github.com/kubernetes/kubernetes/issues/76970

```release-note
NONE
```

/assign @tallclair
/sig node

### CODE REVIEW

File: pkg/apis/core/types.go [hunk 1]
```diff
@@ -2672,7 +2672,7 @@ type PodSpec struct {
 	// If unset or empty, the "legacy" RuntimeClass will be used, which is an implicit class with an
 	// empty definition that uses the default runtime handler.
 	// More info: https://git.k8s.io/enhancements/keps/sig-node/runtime-class.md
-	// This is an alpha feature and may change in the future.
+	// This is a beta feature.
```
Comment (tallclair) [inline pkg/apis/core/types.go]:
```suggestion
	// This is a beta feature as of Kubernetes v1.14.
```

**Review by tallclair — COMMENTED** [review]
Thanks for fixing this! Sorry to make you regenerate...

File: pkg/apis/core/types.go (hunk 1 above)
Comment (egernst) [reply pkg/apis/core/types.go]: ACK - updated.

**Comment by liggitt** [comment]
cc @tallclair

**Pull request merged**